spk/salt-minion/src/requirements.txt

```diff
@@ -1,6 +1,11 @@
-certifi==2020.6.20
+# https://github.com/saltstack/salt/blob/master/requirements/static/pkg/py3.8/linux.txt
+certifi==2021.5.30
```

Comment:
again, to rename as `requirements-pure.txt` and certifi already provided by python310